src/pybind/rados/rados.pyx

```diff
@@ -1870,7 +1870,7 @@ cdef class WriteOp(object):
             uint64_t _offset = offset
 
         with nogil:
-            rados_write_op_zero(self.write_op, _length, _offset)
+            rados_write_op_zero(self.write_op, _offset, _length)
```

Comment:
A test in `src/test/pybind/test_rados.py` would be nice.

A `PendingReleaseNote` is also appropriate. Any users of this library need to audit their code for bugs (like reversed arguments to this `zero` method).